src/layouts/components/Editor/components/BlockWrapper/BlockWrapper.tsx

```diff
@@ -7,16 +7,19 @@ import "./styles.scss"
 interface BlockWrapperProps {
   name: string
   isSelected: boolean
+  otherButtons?: JSX.Element
 }
 
 export const BlockWrapper = ({
   name,
   isSelected,
+  otherButtons,
   children,
 }: PropsWithChildren<BlockWrapperProps>): JSX.Element => {
   return (
     <Box as={NodeViewWrapper}>
       <Box position="relative" maxW="36.5rem" data-group>
+        {isSelected && <>{otherButtons}</>}
```

Comment:
could i ask why we require this? i think an alternative that we might wish to consider here would be to create another component that uses `BlockViewWrapper` and does this functionality. the reason for doing so is that this `otherButtons` is kind of a unique use case and kinda pollutes the responsibility of `BlockWrapper` (thus tempting other devs to add mroe things here)

if that doesn't work well, i'm ok w/ this if we have a better fitting name

Reply:
Well, ideally we should be using bubble menu for this, but somehow I couldnt get the bubble menu to work to target the component as a whole, it targets the individual cells which does not fit our requirement. 

I am not sure if I understood your suggestion correctly, if I were to create another component wouldnt that copy over quite a bit of code ah? Maybe can quick pair if you have bandwidth please!

Reply:
if it's the bubble menu, wouldn't it be possible to check if the node that's active is the `detailSummary`?

Reply:
Hmm the issue is not to check whether the node is active, the issue was revolving more around `detailSummary` being selected but the popper js targeting each detail summary rather than the entire detail group